app/cli/src/main/scala/org/bitcoins/cli/Cli.scala

```diff
@@ -205,6 +359,34 @@ object Cli extends App {
   }
 
   val requestParam: RequestParam = config.command match {
+    // DLC
+    case CreateDLCOffer(amount,
+                        oracleInfo,
+                        contractInfo,
+                        feeRateOpt,
+                        locktime,
+                        refundLT,
+                        escaped) =>
+      RequestParam(
+        "createdlcoffer",
+        Seq(
+          up.writeJs(amount),
+          up.writeJs(oracleInfo),
+          up.writeJs(contractInfo),
+          up.writeJs(feeRateOpt.getOrElse(SatoshisPerVirtualByte.one)),
```

Comment:
Why is the fee rate optional? Seems dangerous

Reply:
What is the danger? At work it takes a while for the tx to be confirmed